src/main/java/edu/kis/powp/jobs2d/drivers/visitor/IDriverVisitor.java

```diff
@@ -0,0 +1,25 @@
+package edu.kis.powp.jobs2d.drivers.visitor;
+
+import edu.kis.powp.jobs2d.drivers.*;
+import edu.kis.powp.jobs2d.drivers.adapter.LineDriverAdapter;
+import edu.kis.powp.jobs2d.drivers.transformators.TransformingJob2dDriverDecorator;
+
+public interface IDriverVisitor {
+
+    int getCount();
```

Comment:
This method should not be a part of the general Visitor interface. It is specific only for  DriverCountingVisitor